### PR TITLE
Profile section

### PR DESCRIPTION
**Pull request by codinBabe**

### CODE REVIEW

File: profile.css [hunk 1]
```diff
@@ -1 +1,69 @@
-// style goes here
+/* style goes here */
+:root{
```
Comment (emmsdan) [inline profile.css:2]: nice use of css variables.
though i recommend not using Uppercase case letters, a simple `--kabab-case` would be enough and make it consistent

File: profile.css [hunk 2]
```diff
@@ -1 +1,69 @@
...
+input{
+  width: 100%;
+  padding: 12px;
+  margin: 0.8em 0 2.5em 0;
+  display: inline-block;
+  border-radius: 8px;
+  border: 1px solid var(--Gray-5);
+  opacity: 0.8;
+  background: var(--Gray-6);
+  box-shadow: 0px 1px 4px 0px rgba(255, 255, 255, 0.69);
```
Comment (emmsdan) [inline profile.css:44]: be consistent with using css variables, make it cleaner.. good work

File: index.html [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
+  <aside> sider bar here </aside>
+  <main>
+    <form action="">
+      <div class="container">
+        <div class="heading">
+          <h1>Fill in your profile form details</h1>
+          <p>Kindly ensure all details given are correct</p>
+        </div>
+        <div class="row">
+          <div class="col">
```
Comment (emmsdan) [inline index.html:24]: not sure how this will turn out, but I think using two `col` class and each have 4 elements can cause unintentional bug, on mobile.

**Review by emmsdan — APPROVED** [review]
Nice work

File: profile.css (hunk 2 above)
Comment (codinBabe) [reply profile.css:44]: Noted sir.

File: index.html (hunk 3 above)
Comment (codinBabe) [reply index.html:24]: Should I adjust it?

File: profile.css (hunk 1 above)
Comment (codinBabe) [reply profile.css:2]: Will work on it.
Thank you for the feedback

**Pull request merged**